.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
```

```suggestion
    directory: "/"
    labels: ["A2-insubstantial", "B0-silent", "C1-low"]
```

Also, if we don't add a `labels` entry, it will add a `rust` label, which we probably don't want.
It will automatically add a `rust` label